atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaTest.java

```diff
@@ -209,10 +214,31 @@ private void checkIfFilesAreTheSame(List<String> generatedTestTables) {
         generatedTestTables.forEach(tableName -> {
             String generatedFilePath =
                     String.format("com/palantir/atlasdb/table/description/generated/%s.java", tableName);
-
             File expectedFile = new File(EXPECTED_FILES_FOLDER_PATH, generatedFilePath);
             File actualFile = new File(testFolder.getRoot(), generatedFilePath);
-            assertThat(actualFile).hasSameContentAs(expectedFile);
+
+            assertThat(expectedFile.length()).isEqualTo(actualFile.length());
+
+            try (Stream<String> expectedFileStream = java.nio.file.Files
+                        .lines(Paths.get(EXPECTED_FILES_FOLDER_PATH, generatedFilePath));
+                Stream<String> actualFileStream = java.nio.file.Files
+                        .lines(Paths.get(testFolder.getRoot().getPath(), generatedFilePath));
+                Stream<Boolean> zipped = Streams.zip(
+                        expectedFileStream,
+                        actualFileStream,
+                        (first, second) -> {
+                            if (first.equals(second))
+                                return true;
+                            return first.contains("__CLASS_HASH") && second.contains("__CLASS_HASH");
+                        })) {
+
+                assertThat(zipped
+                        .anyMatch(elem -> !elem))
+                        .isFalse();
```

Comment:
AssertJ does have pretty nice support for streams, so you can do something like `assertThat(zipped).allMatch(elem -> elem);` directly

Also note that we're asserting `not exists not elem`, which is equivalent to `for-all elem` which I think is clearer